tests/integration/genutil/gentx_test.go

```diff
@@ -316,3 +315,12 @@ func TestDeliverGenTxs(t *testing.T) {
 		})
 	}
 }
+
+func assertNotPanics(t *testing.T, f func()) {
```

Comment:
as discussed, best to have the utils function in the SDK, so we can use them everywhere.